src/ReasonApolloTypes.re

```diff
@@ -8,7 +8,7 @@ type queryString;
  * query string to the standard GraphQL AST.
```

Comment:
As there are no actual changes in this file, just refmt, it should probably not be in this PR.